src/components/faceVerification/screens/IntroScreen.jsx

```diff
@@ -194,6 +241,26 @@ const IntroScreen = ({ styles, screenProps, navigation }) => {
     )
   }
 
+  if (disposing !== false) {
+    return (
+      <View display="flex" justifyContent="center">
+        <ActivityIndicator size="large" />
+      </View>
+    )
+  }
+
+  if (isCitizen) {
```

Comment:
the correct check would be to see if the user has a "lastAuthenticated" >0
because a user can lose his whitelisted state (ie no longer isCitizen) but this is still his second time.
also if someone is a citizen he will not reach the FV screen, unless its part of the goodid upgrade.

Reply:
using the useIdentity hook from sdk-v2 which works for both wallet/dapp.

added here: https://github.com/GoodDollar/GoodDAPP/pull/4303/commits/ee5fe796107d333c85e604d9d024a251edef569e